prisma/schema.prisma

```diff
@@ -15,7 +15,7 @@ datasource db {
 // ===================User Model==================
 model User {
   id        Int     @id @default(autoincrement())
-  uuid      String  @unique @default(uuid())
+  uuid      String  @unique @default(cuid())
```

Comment:
lets not change uuid to cuid for now.
once we do lets do for all the tables

won't this bring a issue in our existing db

Reply:
cuid reverted to uuid for now until all the db are to be changed synchronously